visualize/nopVisualizer.go

```diff
@@ -29,7 +29,9 @@ package visualize
 import (
 	"time"
 
+	energy "github.com/openthread/ot-ns/energy_analyser"
```

Comment:
It does, but because the folder had a different name, Golang's helper modules were automatic changing these imports as I saved that file.
I renamed the folder to avoid this "energy" naming in front of the import.